README.md

```diff
@@ -30,7 +30,7 @@ After installing the requirements, run `yarn && yarn dev` to start developing.
 
 - `yarn build`: Build all packages
 - `yarn clean`: Remove all `node_modules`, `.turbo`, and build folders, clear Yarn cache
-- `yarn dev`: Develop all packages and examples
+- `yarn dev`: Builds all packages and examples and watches for changes, which will trigger rebuilds
```

Comment:
this just builds packages/react-sdk, then runs the `dev` command in all other packages and examples. currently, it builds the SDK, runs the example app in dev mode, and starts dev mode on the SDK.